<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDevice</class>
 <widget class="QDialog" name="FrmDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="HorLayoutDevice">
     <item>
      <widget class="QComboBox" name="CBoxProtoType"/>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddDevice">
       <property name="text">
        <string>AddDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddByHand">
       <property name="text">
        <string>AddByHand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnModifNet">
       <property name="text">
        <string>ModifNet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnUpgrade">
       <property name="text">
        <string>Upgrade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnFacInfo">
       <property name="text">
        <string>FacInfo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="TviewSearchDevice"/>
   </item>
   <item row="5" column="0">
    <widget class="QTableView" name="TviewChnDevice"/>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="HorLayoutChnDevice">
     <item>
      <widget class="QPushButton" name="BtnChnDeleteDevice">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnChnModifDevice">
       <property name="text">
        <string>ModifDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="WidGetTitle" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>33</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QWidget" name="MenuWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>33</height>
         </size>
        </property>
        <widget class="QPushButton" name="BtnMenu">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>32</width>
           <height>33</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Menu</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="BtnMin">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>0</y>
           <width>32</width>
           <height>33</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="BtnMax">
         <property name="geometry">
          <rect>
           <x>64</x>
           <y>0</y>
           <width>32</width>
           <height>33</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="BtnClose">
         <property name="geometry">
          <rect>
           <x>96</x>
           <y>0</y>
           <width>40</width>
           <height>33</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="LabTitle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Devices</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="LabIco">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="ProgressSearch">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>SearchProgress</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
